src/trace_comparison.rs

```diff
@@ -248,6 +257,63 @@ fn compare_storage_entries(base_diffs: Value, native_diffs: Value) -> Value {
     Value::Array(output)
 }
 
+/// Compares the trace of two block reports.
+/// If the trace roots are different, it compares the contract dependencies and storage dependencies.
+/// Otherwise, it skips comparison for contract dependencies and storage dependencies.
+fn compare_trace(base_trace: Value, native_trace: Value) -> Value {
```

Comment:
@rodrigo-pino I remember that there were some additional fields to ignore in `trace_root`, but I can't remember what they were

Reply:
I just pushed some changes with them. If you update from main, you'll see them